README.md

```diff
@@ -27,14 +27,14 @@
 
 ## Motivation
 
-**modern & fast**:
-The project focuses on best practices for e-commerce, like: SSR & ISR, image optimization.
+🏎️ **modern & fast**:
+The project utilizes all the best practices for e-commerce, like SSR, ISR, and image optimization
 
-**easily customizable**:
+🛠️ **easily customizable**:
 TailwindCSS can be easily extended and tweaked, or completely replaced with your favorite CSS solution.
 
-**works out-of-the-box**:
-Pre-configured tools for DX.
+👌 **works out-of-the-box**:
+Pre-configured tools built with DX in mind.
 
 ## Stack
```

Comment:
I thought quite a lot about it and decided to skip this information in README, at least in the context of "bragging" about the core qualities of Storefront (like here).

I understand that both pnpm and Turborepo are cool technologies that may catch people's attention, but these are just tools we used to develop the app that may even change over time. I would go as far as removing the "Stack" section from the README because I think while making the first impression, we should focus on what problems this repository solves, not how.

Reply:
All right... but we should still have this information *somewhere* - it's pretty important part of documentation, usefull when debugging issues or adding new features.

To give you an example: Turborepo caches builds. Caching depends on the configuration stored in repository in `turbo.json`, in there you need to specify what causes to "bust" the cache. It's the file output from `build` command + env variables.

Here's the main source of issues I had so far: you forgot to add that env variable to `turbo.json` and 💥 you are wondering for hours why your production didn't change 😅

I wouldn't say it's **bragging** (maybe a litte bit 😎), no less than *Motivation* paragraph.

We also shouldn't assume people are familiar with those technologies - sure you can look stuff up from `package.json`, but what's that `app` and `packages` folders doing there... that's harded to figure out if you didn't have experience with monorepos.

Reply:
Ah, yes. I see what you mean. I definitely agree that we need to mention Turborepo, I just noticed I completely removed it, even from `docs/development.md`. My main point was that I don't think `README.md` should be the place where we focus on the technologies.

So what I suggest is the following:
1. I will add `Turborepo` to the "Stack" section in `README.md`.
2. I will cover it in greater detail in `docs/development.md`. 

What do you think?